NumLib/Assembler/VectorMatrixAssembler.h

```diff
@@ -32,27 +31,28 @@ getRowColumnIndices(std::size_t const id,
         indices.insert(indices.end(), idcs.begin(), idcs.end());
     }
 
-    return NumLib::LocalToGlobalIndexMap::RowColumnIndices(indices,
-                                                                 indices);
+    return indices;
```

Comment:
why not rename this function if it will no longer return row and column indices?